src/index.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
 );
 
```

Comment:
- Kindly refactor the ` index.js` to be the way that it was before. it must import the `App.js` component and render it to the ReactDOM.
- Only the `App.js` component should ever be rendered in the `index.js` file. All other components should be routed in the `App.js` main component.